minion/minion-grpc/grpc-client/src/test/java/org/opennms/horizon/minion/grpc/channel/SSLChannelFactoryTest.java

```
@@ -109,9 +110,12 @@ void testEmptyTrustStoreWithException() throws Exception {
         channelFactory.setTrustStoreType("pkcs12");
         channelFactory.setTrustStorePassword("changeit");
         when(keyStoreFactory.createKeyStore("pkcs12", trustStore, "changeit")).thenThrow(new GeneralSecurityException(""));
-        when(channelBuilderFactory.create(eq("baz"), eq(443), isNull(), any(TlsChannelCredentials.class))).thenReturn(managedChannelBuilder);
 
-        channelFactory.create("baz", 443, null);
+        try {
+            channelFactory.create("baz", 443, null);
+        } catch (Exception e) {
+
```

Since we are swallowing the exception here... and this seems to be unexpected - why don't we have the test fail if an exception is thrown on create?

---

Updated test.